testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jsp/JspWithUnusedDoubleTestCase.java

```diff
@@ -0,0 +1,44 @@
+package org.jboss.as.test.integration.jsp;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.OperateOnDeployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.test.integration.common.HttpRequest;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.UrlAsset;
+import org.jboss.shrinkwrap.api.exporter.ZipExporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class JspWithUnusedDoubleTestCase {
+    private static final String DEPLOYMENT = "jsp-with-unused-double.war";
+
+    @Deployment(name = DEPLOYMENT)
+    public static WebArchive deployment() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, DEPLOYMENT);
+        war.add(new UrlAsset(JspWithUnusedDoubleTestCase.class.getResource("jsp-with-unused-double.jsp")), "jsp-with-unused-double.jsp");
+        war.as(ZipExporter.class).exportTo(new File("/tmp", war.getName()), true);
```

Comment:
why? for debugging purposes? sure, but don't keep it in testsuite by default it just messes up with CI servers